types/js-sha1/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for js-sha1 0.6
+// Project: https://github.com/emn178/js-sha1
+// Definitions by: Igor N. Dultsev <https://github.com/yhaskell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface JsSha1 {
+    (message: string): string;
+    update(message: string): string;
+    hex(message: string): string;
+    array(message: string): number[];
+    digest(message: string): number[];
+    arrayBuffer(message: string): ArrayBuffer;
+}
+
+declare var sha1: JsSha1;
+
+export = sha1;
```

Comment:
this one looks like old-school UMD module, not CommonJS module:
https://github.com/emn178/js-sha1/blob/master/src/sha1.js
so: https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
Please review the module type

Reply:
the module works as CommonJS and UMD module (see https://github.com/emn178/js-sha1/blob/65b3898c4eecc919477dce9c6e41bb0be1b65219/src/sha1.js#L361). 
I've added an `export as namespace` following @gabritto's advice, so the `export =` also stays, right?

Reply:
yes, thx!